doc/man/zcash-cli.1

```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.3.
-.TH ZCASH-CLI "1" "February 2017" "zcash-cli v1.0.6" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.4.
+.TH ZCASH-CLI "1" "March 2017" "zcash-cli v1.0.7" "User Commands"
 .SH NAME
-zcash-cli \- manual page for zcash-cli v1.0.6
+zcash-cli \- manual page for zcash-cli v1.0.7
 .SH DESCRIPTION
-Zcash RPC client version v1.0.6\-dirty
+Zcash RPC client version v1.0.7\-dirty
```

Comment:
Probably occurs because there were uncommitted changes? Perhaps we could just change release documentation